views/index.html

```diff
@@ -68,4 +75,4 @@
       </div>
     </div>
   </body>
-</html>
+</html>
```

Comment:
Please ensure all files have a newline at the end, see https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file for more info on the reasoning.

Reply:
O.k